tests/test_demography.py

```diff
@@ -1408,28 +1408,28 @@ def test_mean_one_pop(self):
         #  + 400 * exp(-100/200) * (1 - exp(-(300-100)/400))
         #  + 200 * exp(-100/200 - (300-100)/400)
         ddb = self.one_pop_example()
-        m = ddb.mean_coalescence_time(num_samples=[2])
+        m = ddb.mean_coalescence_time(samples=4)
```

Comment:
this is harmless, but so we don't wonder "why 4" maybe this should be put back to `1`.

Reply:
Will do - I stuck this in wondering if the number actually changes anything.